zigbeetuya/integrationpluginzigbeetuya.json

```diff
@@ -100,6 +100,190 @@
                             "defaultValue": 0
                         }
                     ]
+                },
+                {
+                    "id": "f3ce6d68-7c99-47e0-8c2b-17b39ead24e5",
+                    "name": "temperatureSensor",
+                    "displayName": "Temperature sensor",
+                    "interfaces": ["temperaturesensor", "wirelessconnectable", "battery"],
+                    "paramTypes": [
+                        {
+                            "id": "4c850b3c-2bf8-4640-8be9-fbd538c63076",
+                            "name": "ieeeAddress",
+                            "displayName": "IEEE adress",
+                            "type": "QString",
+                            "defaultValue": "00:00:00:00:00:00:00:00"
+                        },
+                        {
+                            "id": "47ec14d4-53be-4d7c-94fb-8d2fe4943cfb",
+                            "name": "networkUuid",
+                            "displayName": "Zigbee network UUID",
+                            "type": "QString",
+                            "defaultValue": ""
+                        },
+                        {
+                            "id": "1f6374ca-11ea-4686-ab9d-067112dd4ea2",
+                            "name": "endpointId",
```

Comment:
we'd only use the endpointId, manufacturer and model parameters in the generic plugin, given the generic plugin handles any compliant device. In the manufacturer specific plugins instead the endpointId parameter is not needed as it's normally well known and just hardcoded in the .cpp (as you did with endpoint 0x01). The manufacturer and model parameters an not needed because in a manufacturer specific plugin the Vendor/ThingClass already would have that information.

So you can just drop those 3 parameters from here.

Reply:
in the Power socket of Tyua  you have also used these parameters.

Reply:
That would be a copy/paste error on my end too, yes. Thanks for pointing it out. Removed the endpointId param in #27, manufacturer and model weren't there tho.